scale/storage/configuration/workspace_configuration.py

```diff
@@ -0,0 +1,71 @@
+"""Defines the configuration for a storage Workspace"""
+from __future__ import unicode_literals
+
+from jsonschema import validate
+from jsonschema.exceptions import ValidationError
+
+from storage.configuration.exceptions import InvalidWorkspaceConfiguration
+
+DEFAULT_VERSION = '1.0'
+
+WORKSPACE_CONFIGURATION_SCHEMA = {
+    'type': 'object',
+    'required': ['broker'],
+    'additionalProperties': False,
+    'properties': {
+        'version': {
+            'description': 'Version of the Workspace configuration schema',
+            'type': 'string',
+            'pattern': '^.{0,50}$',
+        },
+        'broker': {
+            'type': 'object',
+            'required': ['type', 'mount'],
+            'additionalProperties': False,
+            'properties': {
+                'type': {
+                    'type': 'string',
+                    'enum': ['nfs'],
```

Comment:
Right now 'host' is the only valid broker type. I'm currently working on the NFS one.